app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -68,10 +69,16 @@ open class FirefoxApplication : LocaleAwareApplication() {
             initGlean()
             TelemetryIntegration.INSTANCE.init(this)
             initFretboard()
-            serviceLocator.admIntegration.initPush()
 
             enableStrictMode()
 
+            // For now, ignore the violations.
+            // android-components issue #4166:
+            // https://github.com/mozilla-mobile/android-components/issues/4166
```

Comment:
nit: I don't think you need the link & the bug number. I prefer the bug numbers: `a-c#4166`, which is similar to GitHub's cross-repository link syntax. For example: https://github.com/mozilla-mobile/android-components/issues/4166